.github/workflows/luajit-integration.yml

```diff
@@ -45,7 +45,7 @@ on:
 
 jobs:
   luajit-integration:
-    runs-on: ${{ inputs.host }}
+    runs-on: [self-hosted, '${{ inputs.host }}']
```

Comment:
In such cases, we usually do so (add a space after `[` and before `]`):

```yaml
runs-on: [ self-hosted, '${{ inputs.host }}' ]
```

Reply:
Fixed.